docs/kratos/self-service/flows/user-logout.mdx

```diff
@@ -3,115 +3,85 @@ id: user-logout
+Redirects** page using the **Post-Logout Redirect** value.
 
-The following scripts show you how to do the browser flow based on cURL. Prerequisites are a recent version of cURL and a
-registered account in Ory Identities.
+```mdx-code-block
+  </TabItem>
+  <TabItem value="cli" label="Ory CLI">
+```
 
-Before we start, we need to log in:
+You can set the default return URL after a logout using the [Ory CLI](../../../guides/cli/01_installation.mdx):
 
 ```shell
-# Username/email and password for an existing account
-username="myuser@ory.sh"
-password="mysecret"
-
-# We use this cookie jar to initiate the login flow
-cookieJar=$(mktemp)
-
-# Initialize the flow
-flow=$( \
-  curl -s -H "Accept: application/json"  --cookie $cookieJar --cookie-jar $cookieJar \
-    'https://playground.projects.oryapis.com/self-service/login/browser' \
-)
-
-# Get the action URL
-actionUrl=$(echo $flow | jq -r '.ui.action')
-
-# Get the CSRF Token
-csrfToken=$( \
-  echo $flow | \
-    jq -r '.ui.nodes[] | select(.attributes.name=="csrf_token") | .attributes.value' \
-)
-
-# Complete the login
-session=$( \
-  curl -s --cookie $cookieJar --cookie-jar $cookieJar -X POST \
-    -H "Accept: application/json" -H "Content-Type: application/json" \
-    --data '{ "identifier": "'$username'", "password": "'$password'", "method": "password", "csrf_token": "'$csrfToken'" }' \
-    "$actionUrl" \
-)
-
-echo $session | jq
-
-# Check the current user id
-curl -s --cookie $cookieJar --cookie-jar $cookieJar -H "Accept: application/json" \
-  https://playground.projects.oryapis.com/sessions/whoami | \
-  jq -r ".id"
+ory patch identity-config {project-id} \
+   --replace '/selfservice/flows/logout/after/default_browser_return_url="https://example.org"'
 ```
 
-To log out, first get the generated Logout URL then point the Browser to it:
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
 
-```shell
-# Get the Logout URL
-logoutUrl=$( \
-  curl -s --cookie $cookieJar --cookie-jar $cookieJar -H "Accept: application/json" \
-    https://playground.projects.oryapis.com/self-service/logout/browser | \
-    jq -r ".logout_url" \
-)
-
-# Complete the logout
-curl -s --cookie $cookieJar --cookie-jar $cookieJar "$logoutUrl"
-
-# Check the current user id again. It should be `null` after a successful logout
-curl -s --cookie $cookieJar --cookie-jar $cookieJar -H "Accept: application/json" \
-  https://playground.projects.oryapis.com/sessions/whoami | \
-  jq -r ".id"
+If an error occurs, the browser is redirected to the [Error UI](./user-facing-errors.md).
+
+## Single page application (SPAs)
+
+Similar to server-side browser applications, Ory Identities first generates a logout URL for a given Ory Session Cookie. However,
+you can simply call the logout URL using an AJAX request.
+
+The following example requires the [Ory Tunnel](../../../guides/cli/20_proxy-and-tunnel.mdx).
```

Comment:
That's a bit of a surprise. I'd expect that in the context of an app that we're shown and that's just an isolated bit of code just for logout. Can you make it more generic? I can imagine that this confuses someone and makes them thing that you need Tunnel to do logouts in SPAs.

Reply:
Examples from the next paragraph don't say anything like that - that's the approach I'd expect.

Reply:
> makes them thing that you need Tunnel to do logouts in SPAs

You need the tunnel to logout (or rather use Ory Network at all) from SPAs, AFAIU. Because with SPAs you run into CORS issues, where the browser prevents you from talking to a domain apart from the domain the application is hosted on.

Reply:
To elaborate on this: I think they way people will use this doc is, they will implement Ory in their `SPA` (client side javascript!) and at some point want to see how logout is done. 

But if they already use the Ory Network for development (as we intend them to) they already have the `ory tunnel` running, because you can't write SPA apps that interact with the Ory Network without it, AFAICU. This sentence is just here as a reminder, because if you do it without the tunnel, the browser will just throw a bunch of CORS errors (which are really annoying to debug, let me tell you :D) and nothing works.